kafka/config.go

```diff
@@ -2,37 +2,49 @@ package kafka
 
 type Config struct {
 	// A list of Kafka brokers in the form of "host:port"
-	Brokers                []string `mapstructure:"brokers" json:"brokers"`
-	OrderTopic             string   `mapstructure:"order_topic" json:"order_topic"`
-	OrderStatusTopic       string   `mapstructure:"order_status_topic" json:"order_status_topic"`
-	MatchedOrderTopic      string   `mapstructure:"matched_order_topic" json:"matched_order_topic"`
-	OrderBookAdminTopic    string   `mapstructure:"order_book_admin_topic" json:"order_book_admin_topic"`
-	OrderBookStatusTopic   string   `mapstructure:"order_book_status_topic" json:"order_book_status_topic"`
-	OrderBookUpdatesTopic  string   `mapstructure:"order_book_updates_topic" json:"order_book_updates_topic"`
-	OrderBookSnapshotTopic string   `mapstructure:"order_book_snapshots_topic" json:"order_book_snapshots_topic"`
-	AssetsRequestTopic     string   `mapstructure:"assets_request_topic" json:"assets_request_topic"`
-	AssetSnapshotTopic     string   `mapstructure:"assets_snapshot_topic" json:"assets_snapshot_topic"`
-	AssetUpdateTopic       string   `mapstructure:"assets_update_topic" json:"assets_update_topic"`
-	BalancesRequestTopic   string   `mapstructure:"balances_request_topic" json:"balances_request_topic"`
-	BalanceSnapshotTopic   string   `mapstructure:"balances_snapshot_topic" json:"balances_snapshot_topic"`
-	BalanceUpdateTopic     string   `mapstructure:"balances_update_topic" json:"balances_update_topic"`
+	Brokers                   []string `mapstructure:"brokers" json:"brokers"`
+	OrderTopic                string   `mapstructure:"order_topic" json:"order_topic"`
+	OrderStatusTopic          string   `mapstructure:"order_status_topic" json:"order_status_topic"`
+	MatchedOrderTopic         string   `mapstructure:"matched_order_topic" json:"matched_order_topic"`
+	OrderBookAdminTopic       string   `mapstructure:"order_book_admin_topic" json:"order_book_admin_topic"`
+	OrderBookStatusTopic      string   `mapstructure:"order_book_status_topic" json:"order_book_status_topic"`
+	OrderBookUpdatesTopic     string   `mapstructure:"order_book_updates_topic" json:"order_book_updates_topic"`
+	OrderBookSnapshotTopic    string   `mapstructure:"order_book_snapshots_topic" json:"order_book_snapshots_topic"`
+	AssetsRequestTopic        string   `mapstructure:"assets_request_topic" json:"assets_request_topic"`
+	AssetSnapshotTopic        string   `mapstructure:"assets_snapshot_topic" json:"assets_snapshot_topic"`
+	AssetUpdateTopic          string   `mapstructure:"assets_update_topic" json:"assets_update_topic"`
+	BalancesRequestTopic      string   `mapstructure:"balances_request_topic" json:"balances_request_topic"`
```

Comment:
Do we need the balance topics, aren't they being replaced by the User Ledger and Pool Balances?

Reply:
Yea, probably we don't

Reply:
Deletes those ^